### PR TITLE
Food consumption

### PR DESCRIPTION
**Pull request by Andrewwelton**
pls to be reviewing kthx

### CODE REVIEW

File: App/Containers/DrawerContent.js [hunk 1]
```diff
@@ -40,14 +40,26 @@ class DrawerContent extends Component {
...
     return (
       <ScrollView style={styles.container}>
         <DrawerButton text={I18n.t('change_cat')} onPress={this.handlePressChangeCat} />
+        <DrawerButton text={I18n.t('food_consumption')} onPress={this.handlePressFoodConsumption} />
         <DrawerButton text={I18n.t('logOut')} onPress={this.handlePressLogOut} />
+        {/* <DrawerButton text='Component Examples' onPress={this.handlePressComponents} />
+        <DrawerButton text='Usage Examples' onPress={this.handlePressUsage} />
+        <DrawerButton text='API Testing' onPress={this.handlePressAPI} />
+        <DrawerButton text='Themes' onPress={this.handlePressTheme} />
+        <DrawerButton text='Device Info' onPress={this.handlePressDevice} /> */}
```
Comment (melneubert) [inline App/Containers/DrawerContent.js]: Why dis? 😢

File: App/Containers/Dashboard.js [hunk 2]
```diff
@@ -199,7 +217,8 @@ export default class Dashboard extends React.Component {
       )
     }
 
-    if (this.state[StorageKeys.FITBIT_ACCESS_TOKEN] !== null && this.state[StorageKeys.CAT_ID] !== null) {
+    if ((this.state[StorageKeys.FITBIT_ACCESS_TOKEN] !== null && this.state[StorageKeys.FITBIT_ACCESS_TOKEN] !== '') &&
+    (this.state[StorageKeys.CAT_ID] !== null && this.state[StorageKeys.FITBIT_ACCESS_TOKEN] !== '')) {
```
Comment (melneubert) [inline App/Containers/Dashboard.js]: Nitpick: if we're going to use these conditions a lot maybe make a function? I know they're used below as well.

**Review by melneubert — APPROVED** [review]
👍

File: App/Containers/FoodConsumption.js [hunk 3]
```diff
@@ -0,0 +1,167 @@
...
+      formBrandText: '',
+      formDescText: ''
+    })
+  }
+
+  buildPicker = () => {
+    var foodTypes = this.state.foodTypes
+    if (foodTypes === null || foodTypes.length === 0) {
+      return (
+        <Picker.Item label='No saved food types' value={{}} />
```
Comment (ipavl) [inline App/Containers/FoodConsumption.js]: I18n

File: App/Containers/FoodConsumption.js [hunk 4]
```diff
@@ -0,0 +1,167 @@
...
+        <ScrollView style={styles.container}>
+          <TouchableOpacity style={styles.addFoodTypeContainer} onPress={this.handleAddFoodTypePress}>
+            <Text style={styles.sectionText}>Add a new food type</Text>
+            {this.state.hideFoodForm ? <Icon style={styles.addFoodTypeIcon} name='plus' size={20} /> : <Icon style={styles.addFoodTypeIcon} name='minus' size={20} />}
+          </TouchableOpacity>
+          {this.state.hideFoodForm ? null
+            : <View style={styles.paddedContainer}>
+              <TextInput
+                style={{height: 40, borderColor: 'gray', borderWidth: 1, paddingLeft: 10, marginBottom: 15}}
+                placeholder='Name (Required)'
```
Comment (ipavl) [inline App/Containers/FoodConsumption.js]: I18n and below x2

File: App/Containers/ViewFoodConsumption.js [hunk 5]
```diff
@@ -0,0 +1,75 @@
...
+          </View>
+        )
+      }
+      return (
+        <View key={index}>
+          <Text style={styles.labelText}>
+            {date}
+          </Text>
+          <Text style={styles.valueText}>
+            No food recorded.
```
Comment (ipavl) [inline App/Containers/ViewFoodConsumption.js]: I18n

File: App/Containers/ViewWaterConsumption.js [hunk 6]
```diff
@@ -0,0 +1,69 @@
...
+          </View>
+        )
+      }
+      return (
+        <View key={index}>
+          <Text style={styles.labelText}>
+            {date}
+          </Text>
+          <Text style={styles.valueText}>
+            No water consumption recorded.
```
Comment (ipavl) [inline App/Containers/ViewWaterConsumption.js]: I18n

File: App/Containers/WaterConsumption.js [hunk 7]
```diff
@@ -0,0 +1,61 @@
...
+    WellCatManager.submitWaterRecord(currentPetId, inputWater)
+  }
+
+  render () {
+    return (
+      <View style={styles.mainContainer}>
+        <ScrollView style={styles.container}>
+          <View style={styles.paddedContainer}>
+            <Text style={styles.sectionText}>
+              How many cups did your cat drink?
```
Comment (ipavl) [inline App/Containers/WaterConsumption.js]: I18n

File: App/Containers/DrawerContent.js (hunk 1 above)
Comment (Andrewwelton) [reply App/Containers/DrawerContent.js]: Didn't realize those had been removed. Will kill.

**Review by ipavl — APPROVED** [review]

**Pull request merged**